cscs-checks/prgenv/src/gpu_omp_bind_cores.txt

```diff
@@ -0,0 +1,24 @@
+Tag: affinity/affinity, Hostname: nid00001, Rank: 0, Thread: 0
```

Comment:
How come you have several threads, if the code is not OpenMP enabled?

Reply:
Same as above, `OPENMP ?= 1 ` on the Makefile.